react/features/invite/functions.ts

```diff
@@ -420,20 +434,32 @@ export function invitePeopleAndChatRooms(
         inviteServiceUrl: string,
         inviteUrl: string,
         jwt: string,
-        inviteItems: Array<Object>
+        inviteItems: Array<Object>,
+        peopleSearchTokenLocation?: string,
+        peopleSearchTokenKey?: string,
 ): Promise<any> {
 
     if (!inviteItems || inviteItems.length === 0) {
         return Promise.resolve();
     }
 
+    // Parse all the query strings of the search directory endpoint
+    const params = new URLSearchParams();
+    params.append('jwt', jwt);
+
+    // Authentication params for external entities (e. g. email)
+    if (peopleSearchTokenLocation && peopleSearchTokenKey) {
+        const peopleSearchToken = localStorage.getItem(peopleSearchTokenLocation) ?? "";
+        params.append(peopleSearchTokenKey, peopleSearchToken);
```

Comment:
Why do you need a second token parameter. Can't you use Authorization': `Bearer  headers that are added.

Reply:
Can you give more details on this one? 
So the token is stored in localstorage? Who is storing it there?

Reply:
That is a good question. The whole idea is to be able to authenticate against directories on instances where JWT is not enabled. In our case we added a little fetch request to our auth service at the bottom of index.html which retrieves the token if there is an active session.

What we added to index.html is very straightforward and looks like this:

```
document.addEventListener("DOMContentLoaded", () => {
  fetch("{{SERVER}}/api/login/token", { credentials: "include" })
    .then((response) => response.json())
    .then((data) => localStorage.setItem("service_token", data.token));
});
```

Reply:
Do you see a problem with using 'Authorization': `Bearer ${jwt}` with the token you store in service_token?
The idea is to get rid of the URL params with authorization data, as normally the URLs are logged in different places in the backend and is hard to sanitize, so its a better practice to use the headers.

Reply:
Thank you.

Reply:
Use local storage like this: https://github.com/jitsi/jitsi-meet/blob/b4a5e63d1da1e76a7b9abc4f65cb393f42c5bd35/conference.js#L364

Reply:
All good and retested!